/* let a = 1;
let b;
try {
  console.log(b / a);
} catch {
  console.log(a);
} */
console.log("Hello");
console.log("Hello");
console.log("Hello");
console.log("Hello");

try {
  console.log(a);
} catch (e) {
  console.log("Caught Error for a");
  console.log(e);
}
